react/remote-work-demo/src/components/ConversationListItem.jsx

```diff
@@ -1,8 +1,14 @@
 import ConversationImage from "./ConversationImage"
 
-const ConversationListItem = ({conversation, changeConversation, unreadMessages, currentConversation}) => {
-    
-    const unread = unreadMessages.find(item => item.lastMessage.conversation.id === conversation.id);
+const ConversationListItem = ({
+  conversation,
+  changeConversation,
+  unreadMessages,
+  currentConversation,
+}) => {
+  const unread = unreadMessages.find(
+    (item) => item.conversation.id === conversation.id
```

Comment:
Just a passing comment, Egbert mentioned that `lastMessage` was deprecated but that's not the case.